src/Files.App/UserControls/NavigationToolbar.xaml.cs

```diff
@@ -258,32 +258,38 @@ private void ClickablePath_GettingFocus(UIElement sender, GettingFocusEventArgs
 
 		private async void Omnibar_QuerySubmitted(Omnibar sender, OmnibarQuerySubmittedEventArgs args)
 		{
-			if (Omnibar.CurrentSelectedMode == OmnibarPathMode)
+			var mode = Omnibar.CurrentSelectedMode;
```

Comment:
What's the reason to introduce a local variable here for the current mode instance?

Reply:
It's not a major difference, but this helps make the if statements a little cleaner.